src/index.js

```diff
@@ -447,6 +447,7 @@ function createConfig(options, entry, format, writeMeta) {
 						inject: false,
 						extract: options.css !== 'inline',
 						minimize: options.compress,
+						sourceMap: options.sourcemap,
```

Comment:
Think it'd be okay if we disabled CSS sourcemaps for inlined CSS? They increase bundle size a ton because of the Base64 encoding.

```suggestion
						sourceMap: options.sourcemap && options.css !== 'inline',
```

Reply:
good point, deffo want to avoid that